upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -1047,13 +1049,22 @@ func (c *ApplyClusterCmd) validateKubernetesVersion() error {
 	}
 
 	if !util.IsKubernetesGTE(OldestSupportedKubernetesVersion, *parsed) {
+		fmt.Printf("This version of Kubernetes is no longer supported; upgrading Kubernetes is required\n")
+		fmt.Printf("\n")
+		fmt.Printf("More information: %s\n", buildPermalink("upgrade_k8s", OldestRecommendedKubernetesVersion))
```

Comment:
Any reason to use `OldestRecommendedKubernetesVersion` in the url anchor? There isn't an anchor for `#1.10.0` on the page currently.

Reply:
I'm following the example from later in the function where it deprecates or removes support based on what is in the channel. It gives us the flexibility to add version-specific sections to the file at a later date.